tsconfig.json

```diff
@@ -29,7 +29,7 @@
         "alwaysStrict": true,
         "noStrictGenericChecks": true,
         "strictNullChecks": true,
-        "allowSyntheticDefaultImports": false,
+        "allowSyntheticDefaultImports": true,
```

Comment:
What does this change?

Reply:
without this it is impossible to use the default import for packages generated with `esModuleInterop` (which we use in all the projects), not sure why it was disabled (just like the `esModuleInterop` flag - but i kept that as it was, we can switch when converting the codebase to TS)